packages/flutter/lib/src/rendering/object.dart

```diff
@@ -3233,12 +3234,23 @@ abstract class RenderObject with DiagnosticableTreeMixin implements HitTestTarge
   }
 
   /// {@template flutter.rendering.RenderObject.getTransformTo}
-  /// Applies the paint transform up the tree to `ancestor`.
+  /// Applies the paint transform from thie [RenderObject] to the `target`
+  /// [RenderObject].
   ///
   /// Returns a matrix that maps the local paint coordinate system to the
-  /// coordinate system of `ancestor`.
+  /// coordinate system of `target`.
   ///
-  /// If `ancestor` is null, this method returns a matrix that maps from the
+  /// The `skipIfInUnpaintedSubtree` argument determines whether the method
+  /// should return a zero matrix when either this [RenderObject] or `target` is
+  /// in a subtree where the topmost [RenderObject]'s [RenderObject.paintsChild]
+  /// method returns false. In addition to that, this method may also return a
+  /// zero matrix when the paint transform from the common ancestor [RenderObject]
+  /// (of [this] and `target`) to `target` is a singular matrix.
```

Comment:
What does dartdoc link to if you do `[this]`? The class doc for RenderObject?

Reply:
Yeah likely but I didn't mean to make that a link. Changed to `this`.